web-ui/src/login/login.css

```diff
@@ -16,130 +16,47 @@
  */
```

Comment:
We decided to not compile the login.css because it needs to be loaded as soon as the html is loaded. This is because otherwise the UX will break when the disclaimer is shown without css. We preferred to not compile with compass, since there didn't seem to have a need for it and we're trying to remove compass. Eventually we can try to do it with webpack, but this is too big for this story. What do you think?